release/tag/cloudbuild.yaml

```diff
@@ -13,169 +13,80 @@
 # limitations under the License.
 
 steps:
+  - name: gcr.io/cloud-builders/git
+    args: ['clone', 'https://github.com/GoogleContainerTools/kpt.git']
+
+  - name: gcr.io/cloud-builders/git
+    dir: 'kpt'
+    args: ['checkout', '${TAG_NAME}']
+
   # Build the image that contains most of the tools we need (e.g. tar,
   # jq, gzip, golang, bash, etc.) to build kpt tarball. This image
   # (kpt-builder) is re-used on most steps.
   - name: 'gcr.io/cloud-builders/docker'
-    args: ['build', '-f', '/workspace/release/Dockerfile.kpt-build', '-t', 'kpt-builder', '.']
+    dir: 'kpt'
+    args: ['build', '-f', '/workspace/kpt/release/Dockerfile.kpt-build', '-t', 'kpt-builder', '.']
 
-  # Generate and package the licenses and necessary source code.
+  # run e2e tests and linting
   - name: 'kpt-builder'
-    args: ['bash', '/workspace/scripts/create-licenses.sh']
```

Comment:
Are we still generating the licenses that we need to package with kpt?

Reply:
Yeah, it is still generated and included in the package. I have an example release with this setup on my fork of the project: https://github.com/mortent/kpt/releases/tag/v0.0.24